apps/api/src/coscrad-cli/export-schemas.cli-command.ts

```diff
@@ -0,0 +1,72 @@
+import { CommandHandlerService } from '@coscrad/commands';
+import { getCoscradDataSchema } from '@coscrad/data-types';
+import { isNonEmptyString } from '@coscrad/validation-constraints';
+import { existsSync, mkdirSync, writeFileSync } from 'fs';
+import { InternalError } from '../lib/errors/InternalError';
+import { CliCommand, CliCommandOption, CliCommandRunner } from './cli-command.decorator';
+
+type ExportSchemasOptions = {
+    directory: string;
+};
+
+// TODO How does this relate to our types in @coscrad/data-types?
+type CoscradDataClassDefinition = {
+    name: string;
+    type: string;
+    meta: unknown;
+    // TODO tighten this up
+    schema: unknown;
+};
+
+@CliCommand({
+    name: 'export-schemas',
+    description: `export COSCRAD data schemas for domain models, view models, commands, or events`,
+})
+export class ExportSchemasCliCommand extends CliCommandRunner {
+    constructor(private readonly commandHandlerService: CommandHandlerService) {
+        super();
+    }
+
+    async run(_passedParams: string[], options: ExportSchemasOptions) {
```

Comment:
is this supposed to be underscored?

Reply:
Is this just saving a placeholder for eventual cli options down the road?

Reply:
It's part of the external interface for `@nest-coammander` and unused in my use case, but good attention to detail, guys!